code/modules/surgery/organs/organ_external.dm

```diff
@@ -85,6 +85,8 @@
 	if(parent && parent.children)
 		parent.children -= src
 
+	parent = null
```

Comment:
Similar to `owner` in its parent type, I don't think this organ should ever be in a `qdel`able state and this still hold a reference, but....